extension-style-app/src/androidTest/java/com/mapbox/maps/testapp/style/expressions/ExpressionTest.kt

```diff
@@ -1263,6 +1263,9 @@ class ExpressionTest : BaseStyleTest() {
   @UiThreadTest
   fun numberFormatTest() {
     val expression = numberFormat(literal(123.456789)) {
+      locale {
```

Comment:
maybe it's related to this PR but should we check other locales?

Reply:
Yeah, we can add parameterized tests for different locale.